src/components/USDC/index.tsx

```diff
@@ -454,15 +467,15 @@ function USDC() {
       );
       const maxSwap = await contract.calculateMaxSwapAmount(targetAsset);
       if (cancelled) return;
-      setMaxSwapAmount(maxSwap.toBigInt());
+      dispatch(setMaxSwapAmount(maxSwap.toString()));
```

Comment:
redux can't handle the `bigint` type, because it's non-serializable, so we store as a string instead